integration_tests/tests/forseti_integration_testing/controls/inventory.rb

```diff
@@ -0,0 +1,493 @@
+require 'json'
+control 'inventory' do
+
+     describe "Create an inventory" do
+        subject do
+            command("forseti inventory list")
+        end
+
+        before do
+            command("forseti inventory create").result
+        end
+
+        its("stdout") { should match /SUCCESS/}
+
+        after do
+            command("forseti inventory purge 0").result
+        end
+    end
+
+    describe "Create an inventory and check database" do
+        subject do
+            command("mysql -u root --host 127.0.0.1 --database forseti_security --execute \"select count(DISTINCT resource_id) from gcp_inventory where category='resource' and resource_type = 'project';\"")
+        end
+
+        before(:context) do
+            command("forseti inventory create").result
+            command("sudo apt-get -y install mysql-client").result
+        end
+
+        # its("stdout") { should match "2" }
```

Comment:
The count here need to be determined from the source config.  Where is the source config where all the resources are defined?

Reply:
Resources are defined in the`main.tf`.